ui/models/job.js

```diff
@@ -63,7 +63,7 @@ export default class JobModel {
 
   static get(repoName, pk, signal) {
     // a static method to retrieve a single instance of JobModel
-    return fetch(`${getProjectUrl(uri)}${pk}/`, { signal }).then(
+    return fetch(`${getProjectUrl(uri, repoName)}${pk}/`, { signal }).then(
```

Comment:
I am in progress of removing the need for the ``repoName`` property.  However, this ``jobs.get`` function DOES still need it.  In Treeherder we get it from the URL as a fallback.  But you won't have that in Perfherder, so it is correct to add it back in at this location.  

What I intend to do soon is to make a ``/jobs/`` endpoint that does not require ``repoName``.  That will be in  ``/webapp/api/jobs`` in the ``JobsViewSet``.  But that viewset only has a ``list`` function.  It needs a ``retrieve`` function.  

So, your options here are to:

1. add this the way you have done
2. Change the https://github.com/mozilla/treeherder/blob/6581c84289b27657bde7b06e3c1087f6f2c64de2/treeherder/webapp/api/jobs.py#L264 function to stop passing in the ``project``
3. Add a ``retrieve`` function here https://github.com/mozilla/treeherder/blob/6581c84289b27657bde7b06e3c1087f6f2c64de2/treeherder/webapp/api/jobs.py#L160

Number 2 might be the easiest.  Then you don't have to worry about a ``repoName`` in Perfherder for these actions.
However, this is still in progress.  The best thing here would be to use the non-project url.  So, on this line, we should change ``getProjectUrl`` to ``getApiUrl``